src/chrome/content/rules/The_Atlantic.com.xml

```diff
@@ -11,70 +9,23 @@
 	* Shows profiles; mismatched, CN: profiles.theatlantic.com
 
 
-	Problematic domains:
-
-		- theatlantic.com ¹
-		- assets.theatlantic.com	(self-signed)
-		- profiles.theatlantic.com ²
-
-	¹ Dropped
-	² Self-signed
-
-
 	Fully covered domains:
-
+		- www.theatlantic.com
 		- cdn.theatlantic.com
 		- assets.theatlantic.com	(→ cdn.theatlantic.com)
-
-
-	Mixed content:
-
-		- Scripts, on:
-
-			- www from cdn ¹
-			- www from admin.brightcove.com ¹
-			- theatlantic.disqus.com
-
-		- css on www from cdn ¹
-
-		- Ads/web bugs, on www from:
-
-			- atlanticmedia.122.2o7.net ¹
-			- api.bizographics.com ¹
-			- ct.buzzfeed.com ²
-			- siterecruit.comscore.com ¹
-			- ad.doubleclick.net ³
-			- pubads.g.doubleclick.net ³
-			- loadus.exelator.com ¹
-			- apis.google.com ¹
-			- widgets.outbrain.com ¹
-			- catrg.peer39.net ⁴
-			- www.reddit.com ¹
-			- b.scorecardresearch.com ¹
-			- cdn.taboolasyndication.com ¹
-			- a.visualrevenue.com ⁵
-
-		- Images on www from cdn ¹
-
-	¹ Secured by us
-	² Unsecurable
-	³ Rule disabled by default
-	⁴ Rule disabled by default <= mismatched cert
-	⁵ Unsecurable <= 503, akamai
-
 -->
-<ruleset name="The Atlantic.com (partial)">
+<ruleset name="The Atlantic.com">
 
 	<!--	Direct rewrites:
 				-->
+	<target host="www.theatlantic.com" />
 	<target host="cdn.theatlantic.com" />
 	<target host="ssl.theatlantic.com" />
```

Comment:
Please note as `Expired` in a [top comment](https://github.com/EFForg/https-everywhere/search?utf8=%E2%9C%93&q=problematic)